pkg/fleet-manager/plugin/render.go

```diff
@@ -55,7 +55,7 @@ func (plugin FleetPluginConfig) StorageNamespace() string {
 	return plugin.Fleet.Namespace
 }
 
-type FleetCluster struct {
+type KubeConfigSecretRef struct {
 	Name       string
```

Comment:
nit: The Name can be removed

Reply:
Currently, it is used solely for rendering and requires the `name` parameter. 

If we remove `name`, we will need to refactor many corresponding functions

However, I think that refactoring them would not be wise.

such as 
https://github.com/kurator-dev/kurator/blob/5de151793e19756831fddc52b986f6b7309f30b5/pkg/fleet-manager/plugin/render.go#L30-L46

Reply:
I did see that, it is unused

Reply:
you are right, it is unused